### PR TITLE
Adding delete of namespaces

### PR DESCRIPTION
**Pull request by paigerube14**
### Description
Deleting namespaces based on a namespace regex string and/or label_selectors 

### Fixes
https://github.com/cloud-bulldozer/kraken/issues/122

### CODE REVIEW

File: scenarios/regex_namespace.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+scenarios:
```
[the remark this answers is not in the document]
Comment (paigerube14) [reply scenarios/regex_namespace.yaml:1]: Updated

File: README.md [hunk 2]
```diff
@@ -44,6 +44,7 @@ Kraken supports pod, node, time/date and [litmus](https://github.com/litmuschaos
 
 - [Cluster Shut Down Scenarios](docs/cluster_shut_down_scenarios.md)
 
+- [Namespace Scenarios](docs/namespace_scenarios.md)
```
Comment (chaitanyaenr) [inline README.md:47]: @paigerube14 we might have missed the docs around the namespace delete scenarios.

Comment (paigerube14) [reply README.md:47]: Added it in!

File: config/config.yaml [hunk 3]
```diff
@@ -22,6 +22,9 @@ kraken:
         -   cluster_shut_down_scenarios:
             - - scenarios/cluster_shut_down_scenario.yml
               - scenarios/post_action_shut_down.py
+        - namespace_scenarios:
```
Comment (chaitanyaenr) [inline config/config.yaml:25]: We might want to fix the spacing to align with other scenarios

**Review by chaitanyaenr — APPROVED** [review]
LGTM

**Pull request merged**